### PR TITLE
Adding support for various fgbio and Picard tools

### PR DESCRIPTION
**Pull request by nh13**
# If this PR is for a new module
 - [x] There is example tool output for tools in the https://github.com/ewels/MultiQC_TestData repository (see https://github.com/ewels/MultiQC_TestData/pull/139)
 - [x] Code is tested and works locally (including with `--lint` flag)
 - [x] `CHANGELOG.md` is updated
 - [x] `docs/modulename.md` is created
 - [x] Everything that can be represented with a plot instead of a table is a plot
 - [x] Report sections have a description and help text (with `self.add_section`)
 - [x] There aren't any huge tables with > 6 columns (explain reasoning if so)
 - [x] Each table column has a different colour scale to its neighbour, which relates to the data (eg. if high numbers are bad, they're red)
 - [x] Module does not do any significant computational work

### CODE REVIEW

File: multiqc/modules/picard/QualityByCycleMetrics.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    # Plot the data and add section
+    pconfig = {
+        'id': 'picard_quality_by_cycle',
+        'title': 'Picard: Mean Base Quality by Cycle',
+        'ylab': 'Mean Base Quality',
+        'xlab': 'Cycle #',
+        'xDecimals': False,
+        'tt_label': '<b>cycle {point.x}</b>: {point.y:.2f} %',
+        'ymax': 100,
```
Comment (ewels) [inline multiqc/modules/picard/QualityByCycleMetrics.py]: As above, from reading the Picard / GATK docs I think it sounds like this is not a percentage?

**Comment by ewels** [comment]
@nh13 I made some changes in 8fd3a4c as I went through the code for the new Picard tools. The main thing I want to check with you is where I have left comments above about percentages / phred scores. Could you confirm that my suspicions are correct here please?

File: multiqc/modules/picard/QualityByCycleMetrics.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    self.write_data_file(all_data, 'multiqc_picard_quality_by_cycle')
+
+    # Plot the data and add section
+    pconfig = {
+        'id': 'picard_quality_by_cycle',
+        'title': 'Picard: Mean Base Quality by Cycle',
+        'ylab': 'Mean Base Quality',
+        'xlab': 'Cycle #',
+        'xDecimals': False,
+        'tt_label': '<b>cycle {point.x}</b>: {point.y:.2f} %',
```
[the remark this answers is not in the document]
Comment (nh13) [reply multiqc/modules/picard/QualityByCycleMetrics.py]: good catch

File: multiqc/modules/picard/QualityByCycleMetrics.py (hunk 1 above)
Comment (nh13) [reply multiqc/modules/picard/QualityByCycleMetrics.py]: copy-paste error

**Comment by nh13** [comment]
LGTM!

**Comment by ewels** [comment]
Thanks!

**Pull request merged**